numpy/core/fromnumeric.py

```diff
@@ -5,15 +5,15 @@
 
 import types
 import warnings
+from functools import reduce
 
 import numpy as np
+from . import _methods
 from .. import VisibleDeprecationWarning
 from . import multiarray as mu
 from . import umath as um
 from . import numerictypes as nt
 from .numeric import asarray, array, asanyarray, concatenate
-from . import _methods
```

Comment:
This moved because your IDE alphabetized it, I'm guessing? Just curious

Reply:
Actually, is this import needed at all now?

Reply:
Yes, for mean, etc.

Reply:
And no, I actually removed it and then had to put it back. I can put it back where it was if it's an issue.

Reply:
It's slightly better to minimize noise in the diffs, but it's not worth your time to make a new commit to fix that